### ISSUE TITLE
duplicate file prevention

### ISSUE THREAD

**Issue by hongcui** [biosemantics/etc-site#203]
I have an additional suggestion related to the capture process and the file manager.  If by mistake the user duplicates the files (which happens often if you don't clear your text capture screen), the moment that you want to delete the duplicate files from the file manager you have to do this one by one and it takes a long time, there is no way to select several and delete them at once.

May I suggest two alternative solutions:
A) If a file is going to be duplicated, could the system give the user a message alerting that a file is about to be duplicated?
B) if the files are duplicated, could you guys make it so the user can delete several files at the same time.

**Comment by rodenhausen** [comment]
Regarding A) What is we clear the textarea after files are created? Is that enough prevention of duplicates? Also, duplicates in the sense of the file content, or duplicates just by file name?

Regarding B) If I remember correctly, I had already looked into the current tree implementation we use to display the files to see if it supports multiple selections out of the box and it does not. I might be wrong but I think it could take more code rewriting to make this feature happen. 

We may want to consider, that if we spent time on doing major code changes on this part, we may at same time want to make a switch from the current file/task manager to something more like the "File Manager replacement suggestion" in http://taxonconceptexplorer.org/pub/Ideas
